app/models/queue_item.rb

```diff
@@ -0,0 +1,16 @@
+class QueueItem < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :video
+
+  delegate :category, to: :video
+  delegate :title, to: :video, prefix: :video
+
+  def rating
+    review = Review.where(user_id: user.id, video_id: video.id).first
```

Comment:
You should use `.find_by` for single record search. `where(...).first` is inefficient.